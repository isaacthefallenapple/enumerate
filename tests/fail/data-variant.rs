use enumerate::Enumerate;

fn main() {
    #[derive(Enumerate)]
    enum Enum {
        A,
        B { id: usize, name: String },
        C,
    }
}
