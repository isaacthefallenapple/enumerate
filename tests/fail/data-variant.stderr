error: Cannot enumerate enum with tuple or struct variant.
 --> $DIR/data-variant.rs:7:9
  |
7 |         B { id: usize, name: String },
  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: Try skipping the variant with the `#[enumerate(skip)]` attribute.
